dev/breeze/pyproject.toml

```diff
@@ -52,23 +52,14 @@ dependencies = [
     # It turns out that when packages are prepared metadata version in the produced packages
     # is taken from the front-end not from the backend, so in order to make sure that the
     # packages are reproducible, we should pin both backend in "build-system" and frontend in
-    # "dependencies" of the environment that is used to build the packages.
-    #
-    # TODO(potiuk): automate bumping the version of flit in breeze and sync it with
-    # the version in the template for provider packages with pre-commit also add instructions in
-    # the source packages explaining that reproducibility can only be achieved by using the same
-    # version of flit front-end to build the package
-    #
+    # "dependencies" of the environment that is used to build the packages
     "flit==3.10.1",
     "flit-core==3.10.1",
     "google-api-python-client>=2.142.0",
     "google-auth-httplib2>=0.2.0",
     "google-auth-oauthlib>=1.2.0",
     "gitpython>=3.1.40",
-    "hatch==1.9.4",
-    # Importib_resources 6.2.0-6.3.1 break pytest_rewrite
-    # see https://github.com/python/importlib_resources/issues/299
-    "importlib_resources>=5.2,!=6.2.0,!=6.3.0,!=6.3.1;python_version<\"3.9\"",
```

Comment:
Is that removal by intend or a mistake?

Reply:
on purpose I noticed `< 3.9` (which is false by default now), But yeah - I should have mentioned that in the commit.

Reply:
Updated commit message.

Reply:
Ah, yeah, we have no Python <3.9 anymore :-D You are right!